src/styles/SearchStyle.css

```diff
@@ -0,0 +1,19 @@
+.searchForm {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+}
```

Comment:
CSS filenames are by convention kebab cased, as should classes be. There is no need to append `Style` on the end.

Also a bit of a suggestion here to make to UX a little better (Ive renamed the classes to correct casing as mentioned above):

```
.search {
    display: flex;
    justify-content: center;
    flex-direction: column;
    align-items: center;
}

.search-form {
    display: flex;
    flex-direction: column;
    max-width: 400px;
}
```
This will centre the search bar and not make it fill the entire screen. Just a suggestion